converter/src/main/java/org/dswarm/converter/flow/TransformationFlow.java

```diff
@@ -166,7 +166,7 @@ public String getScript() {
 			return Observable.error(new DMPConverterException(msg));
 		}
 
-		return apply(Observable.from(tuplesList), false, false).reduce(
+		return apply(Observable.from(tuplesList), false, false, true).reduce(
```

Comment:
It gets difficult to remember what those boolean parameters mean.
An often used readability helper is to use variables whose name reflect the desired behavior. For example

```java
final boolean dontWriteResultToDatahub = false;
final boolean returnJsonToCaller = false;
final boolean enableVersioning = true;
return apply(Observable.from(tuplesList), dontWriteResultToDatahub, returnJsonToCaller, enableVersioning) // 
...
```

That way, no-one has to look into the function to understand what `false, false, true` means.